core/native/src/main/scala/zio/scheduler/SchedulerLive.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2017-2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.scheduler
+
+import zio.ZIO
+import zio.duration.Duration
+import zio.internal.{ Scheduler => IScheduler }
+
+import scala.scalajs.js
```

Comment:
is it correct to import `scalajs` stuff here? (or is that what `native-loop-js-compat` is for?)

Reply:
@sideeffffect - yep, exactly.  We have a compatability shim that presents Scala Native's async capabilities with a scalajs api.